src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,32 @@
-export const PostInfo = () => (
-  <>Put the post here</>
-);
+import './PostInfo.scss';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({ post }) => {
+  const { title, body, comments, user } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{`${title}`}</h3>
+
+        <p>
+          {' Posted by  '}
+
+          <UserInfo user={user} />
+        </p>
+      </div>
+
+      <p className="PostInfo__body">
+        {`${body}`}
+      </p>
+
+      <hr />
+
+      {comments[0]
```

Comment:
```suggestion
      {!!comments.length
```

better to check on comments length